README.md

```diff
@@ -6,10 +6,23 @@ Provide common management and troubleshooting tools for the [Rook Ceph](https://
 
 ## Install
 
-To install the plugin, run:
+### To install the plugin with krew, run:
 
   ```kubectl krew install rook-ceph```
 
+### To install the plugin as script (has auto-completion), run:
```

Comment:
For now I'd suggest we not document this install method, let's just keep the krew install instructions. While it will be more convenient especially for the completion, the krew install instructions are still the simpler method to get going.

Reply:
We could do what kubectl does and have a plugin command that outputs the completion script. E.g., `kubectl rook-ceph completion bash`